### PR TITLE
IntelliSense fixes - PR 13/?

### PR DESCRIPTION
**Pull request by mairaw**
- Adds missing descriptions
- Adds dropped exception conditions during migration
- Style guide fixes
- Minor fixes

Contributes to #1330 and #1358

### CODE REVIEW

File: xml/System.Net.Configuration/SettingsSection.xml [hunk 1]
```diff
@@ -218,9 +212,7 @@
       </ReturnValue>
       <Docs>
         <summary>Gets the configuration element that controls settings for connections to remote host computers.</summary>
-        <value>A <see cref="T:System.Net.Configuration.ServicePointManagerElement" /> object.  
-  
- The configuration element that that controls setting used network performance counters for connections to remote host computers.</value>
+        <value>The configuration element that that controls setting used network performance counters for connections to remote host computers.</value>
```
Comment (rpetrusha) [inline xml/System.Net.Configuration/SettingsSection.xml]:
```suggestion
        <value>The configuration element that that controls settings for connections to remote host computers.</value>
```

File: xml/System.Collections.Generic/SortedDictionary`2.xml [hunk 2]
```diff
@@ -1295,8 +1298,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Returns an enumerator that iterates through a collection.</summary>
+        <returns>An <see cref="T:System.Collections.IEnumerator" /> that can be used to iterate through the collection.</returns>
```
Comment (rpetrusha) [inline xml/System.Collections.Generic/SortedDictionary`2.xml]:
```suggestion
        <returns>An enumerator that can be used to iterate through the collection.</returns>
```

File: xml/System.Collections.Generic/SortedList`2.xml [hunk 3]
```diff
@@ -1618,8 +1623,8 @@
         <ReturnType>System.Collections.Generic.ICollection&lt;TValue&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets an <see cref="T:System.Collections.Generic.ICollection`1" /> containing the values in the <see cref="T:System.Collections.Generic.IDictionary`2" />.</summary>
+        <value>An <see cref="T:System.Collections.Generic.ICollection`1" /> containing the values in the <see cref="T:System.Collections.Generic.IDictionary`2" />.</value>
```
Comment (rpetrusha) [inline xml/System.Collections.Generic/SortedList`2.xml]:
```suggestion
        <value>An object containing the values in the <see cref="T:System.Collections.Generic.IDictionary`2" />.</value>
```

File: xml/System.Linq/ParallelEnumerable.xml [hunk 4]
```diff
@@ -6600,7 +6600,7 @@
         <typeparam name="TResult">The type to filter the elements of the sequence on.</typeparam>
         <param name="source">The sequence whose elements to filter.</param>
```
Comment (rpetrusha) [inline xml/System.Linq/ParallelEnumerable.xml]:
```suggestion
        <param name="source">The sequence with elements to filter.</param>
```

File: xml/System.Linq/ParallelEnumerable.xml [hunk 5]
```diff
@@ -6600,7 +6600,7 @@
...
         <summary>Filters the elements of a ParallelQuery based on a specified type.</summary>
-        <returns>A sequence that contains elements from the input sequence of type .</returns>
+        <returns>A sequence that contains elements from the input sequence of type.</returns>
```
Comment (rpetrusha) [inline xml/System.Linq/ParallelEnumerable.xml]:
```suggestion
        <returns>A sequence that contains elements from the input sequence filtered by type.</returns>
```

File: xml/System.Net.Configuration/SettingsSection.xml [hunk 6]
```diff
@@ -131,9 +129,7 @@
       </ReturnValue>
       <Docs>
         <summary>Gets the configuration element that enables Internet Protocol version 6 (IPv6).</summary>
-        <value>A <see cref="T:System.Net.Configuration.Ipv6Element" />.  
-  
- The configuration element that controls setting used by IPv6.</value>
+        <value>The configuration element that controls setting used by IPv6.</value>
```
Comment (rpetrusha) [inline xml/System.Net.Configuration/SettingsSection.xml]:
```suggestion
        <value>The configuration element that controls the setting used by IPv6.</value>
```

File: xml/System.Net.Configuration/SettingsSection.xml [hunk 7]
```diff
@@ -161,9 +157,7 @@
       </ReturnValue>
       <Docs>
         <summary>Gets the configuration element that controls whether network performance counters are enabled.</summary>
-        <value>A <see cref="T:System.Net.Configuration.PerformanceCountersElement" />.  
-  
- The configuration element that controls setting used network performance counters.</value>
+        <value>The configuration element that controls setting used network performance counters.</value>
```
Comment (rpetrusha) [inline xml/System.Net.Configuration/SettingsSection.xml]:
```suggestion
        <value>The configuration element that controls usage of network performance counters.</value>
```

File: xml/System.Net.Configuration/SettingsSection.xml [hunk 8]
```diff
@@ -317,9 +305,7 @@
       </ReturnValue>
       <Docs>
         <summary>Gets the configuration element that controls the settings used by an <see cref="T:System.Net.WebUtility" /> object.</summary>
-        <value>Returns <see cref="T:System.Net.Configuration.WebUtilityElement" />.  
-  
- The configuration element that controls the settings used by an <see cref="T:System.Net.WebUtility" /> object.</value>
+        <value>The configuration element that controls the settings used by an <see cref="T:System.Net.WebUtility" /> object.</value>
```
Comment (rpetrusha) [inline xml/System.Net.Configuration/SettingsSection.xml]:
```suggestion
        <value>The configuration element that controls the settings used by a <see cref="T:System.Net.WebUtility" /> object.</value>
```

File: xml/System.Net/HttpListener.xml [hunk 9]
```diff
@@ -757,9 +757,7 @@
       <Parameters />
       <Docs>
         <summary>Waits for an incoming request as an asynchronous operation.</summary>
-        <returns>Returns <see cref="T:System.Threading.Tasks.Task`1" />.  
-  
- The task object representing the asynchronous operation. The <see cref="P:System.Threading.Tasks.Task`1.Result" /> property on the task object returns an <see cref="T:System.Net.HttpListenerContext" /> object that represents a client request.</returns>
+        <returns>The task object representing the asynchronous operation. The <see cref="P:System.Threading.Tasks.Task`1.Result" /> property on the task object returns an <see cref="T:System.Net.HttpListenerContext" /> object that represents a client request.</returns>
```
Comment (rpetrusha) [inline xml/System.Net/HttpListener.xml]: At some point, the extended detail (on the Result property) for this and some of the following APIs should be moved to the Remarks section.

**Comment by mairaw** [comment]
Thanks @rpetrusha. This suggestion feature is really neat. I could just batch them all and commit! I'll take care of the other comments now.

**Pull request merged**